content/authors/jose-yusuf/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Jose Yusuf
+type: authors
+---

```

Comment:
Please refer to the latest author info in the guidelines.

Reply:
guideline still showing the format so I don't understand

Reply:
@josephotienoowino 

The format looks like [this](https://raw.githubusercontent.com/section-engineering-education/engineering-education/master/content/authors/Apondi-Ashley/index.md)

Reply:
Thanks